vmm/src/acpi.rs

```diff
@@ -28,6 +24,22 @@ struct PCIRangeEntry {
     _reserved: u32,
 }
 
+#[repr(packed)]
+#[derive(Default)]
+struct MemoryAffinity {
+    pub type_: u8,
+    pub length: u8,
+    pub proximity_domain: u32,
+    _reserved1: u16,
+    pub base_addr_lo: u32,
+    pub base_addr_hi: u32,
+    pub length_lo: u32,
+    pub length_hi: u32,
+    _reserved2: u32,
+    pub flags: u32,
+    _reserved3: u64,
+}
+
```

Comment:
Any plan to support processor affinity ?

Reply:
Yes, this is part of the following PR I'll submit through this branch: https://github.com/sboeuf/cloud-hypervisor-1/tree/add_guest_numa_cpus